model/pytorch/setup.py

```diff
@@ -0,0 +1,59 @@
+import os
+import ast
+from io import open
+
+from setuptools import find_packages, setup
+
+self_path = os.path.dirname(os.path.realpath(__file__))
+
+with open(os.path.join(self_path, 'dffml_model_pytorch', 'version.py'),
+          'r') as f:
+    for line in f:
+        if line.startswith('VERSION'):
+            version = ast.literal_eval(line.strip().split('=')[-1].strip())
+            break
+
+with open(os.path.join(self_path, 'README.md'), 'r', encoding='utf-8') as f:
+    readme = f.read()
+
+INSTALL_REQUIRES = [
+    "torch",
+    "scikit-learn"
+]
+
+setup(
+    name='dffml-model-pytorch',
+    version=version,
+    description='',
+    long_description=readme,
```

Comment:
Ah, I forgot to update the `skel/` projects (#45)

Can you please add `long_description_content_type='text/markdown',` like in `model/tensorflow`:

https://github.com/intel/dffml/blob/c0ec62c4892b2c1a6ee0362861acd5e05ae717ab/model/tensorflow/setup.py#L27-L29